htdocs/core/class/html.form.class.php

```diff
@@ -4024,13 +4024,8 @@ function load_tva($htmlname='tauxtva', $selectedrate='', $societe_vendeuse='', $
             return $return;
         }
 
-        //var_dump($societe_acheteuse);
-        //print "name=$name, selectedrate=$selectedrate, seller=".$societe_vendeuse->country_code." buyer=".$societe_acheteuse->country_code." buyer is company=".$societe_acheteuse->isACompany()." idprod=$idprod, info_bits=$info_bits type=$type";
-        //exit;
-
-        // Define list of countries to use to search VAT rates to show
-        // First we defined code_country to use to find list
```

Comment:
It may be a good idea to remove "dead code" but not to remove comment. It would be better to complete the comment.